centreon/www/front_src/src/Dashboards/api/endpoints.ts

```diff
@@ -38,7 +38,7 @@ export const dashboardsContactsEndpoint = `${baseEndpoint}/configuration/dashboa
 
 export const dashboardsContactGroupsEndpoint = `${baseEndpoint}/configuration/dashboards/contactgroups`;
 
-export const playlistsEndpoint = `${baseEndpoint}/configuration/dashboards/playlists`;
```

Comment:
can you please the "baseEnpoint" section [here ](https://github.com/centreon/centreon/blob/develop/centreon/www/front_src/src/Dashboards/components/DashboardPlaylists/PlaylistsListing/api/endpoints.ts#L4)

Reply:
You no longer need to use the baseEndpoint